src/stepFunctions/commands/downloadStateMachineDefinition.ts

```diff
@@ -31,15 +32,25 @@ export async function downloadStateMachineDefinition(params: {
             params.stateMachineNode.details.stateMachineArn
         )
 
-        const wsPath = vscode.workspace.workspaceFolders ? vscode.workspace.workspaceFolders[0].uri.fsPath : '/'
-        let filePath = path.join(wsPath, params.stateMachineNode.details.name + '.asl.json')
-        const fileInfo = await vscode.window.showSaveDialog({ defaultUri: vscode.Uri.file(filePath) })
-        if (fileInfo) {
-            filePath = fileInfo.fsPath
-            fs.writeFileSync(filePath, stateMachineDetails.definition, 'utf8')
-            const openPath = vscode.Uri.file(filePath)
-            const doc = await vscode.workspace.openTextDocument(openPath)
-            vscode.window.showTextDocument(doc)
+        if (params.isDownloadAndRender) {
+            const doc = await vscode.workspace.openTextDocument({
```

Comment:
Just making sure: this won't download the definition to a file, it'll just open as a temp file? Nit would be to rename the param to make this more clear.

Reply:
Yes, this is a temporary file.

Reply:
I renamed the param to `isPreviewAndRender`